chainer/_runtime_info.py

```diff
@@ -0,0 +1,36 @@
+import sys
+
+import numpy
+import six
+
+import chainer
+from chainer.backends import cuda
+
+
+class _RuntimeInfo(object):
+
+    chainer_version = None
+    numpy_version = None
+    cuda_info = None
+
+    def __init__(self):
+        self.chainer_version = chainer.__version__
+        self.numpy_version = numpy.__version__
+        self.cuda_info = cuda.get_runtime_info()
+
+    def __str__(self):
+        s = six.StringIO()
+        s.write('''Chainer: {}\n'''.format(self.chainer_version))
```

Comment:
Are there any reasons to use `'''` instead of  `'`?

Reply:
Because these are strings meant to be presented to user, which means they can reasonably contain `'`.